README.md

```diff
@@ -1,5 +1,8 @@
 [![Open in GitPod](https://img.shields.io/badge/Gitpod-Ready--to--Code-blue?logo=gitpod)](https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree) ![Uptime](https://img.shields.io/endpoint?url=https%3A%2F%2Fraw.githubusercontent.com%2FEddieHubCommunity%2Fmonitoring%2Fmaster%2Fapi%2Flink-free-linkfree-io%2Fuptime.json) [![License: MIT](https://img.shields.io/badge/License-MIT-yellow.svg)](https://opensource.org/licenses/MIT) [![GitHub release (latest by date)](https://img.shields.io/github/v/release/EddieHubCommunity/LinkFree)](https://github.com/EddieHubCommunity/LinkFree/releases) ![GitHub repo size](https://img.shields.io/github/repo-size/EddieHubCommunity/LinkFree)
 
+![2](https://github.com/BSatishSutar/LinkFree/assets/40925459/cd8b1c4d-9d4d-420d-8cc5-7d3e00c200ec)
+
+
```

Comment:
I am removing this for the same reasons as I explained on: https://github.com/EddieHubCommunity/LinkFree/pull/8761#issuecomment-1692933613

```suggestion
```